### ISSUE TITLE
Navigable Tasks

### ISSUE THREAD

**Issue by luckysharms9** [ResearchStack/ResearchStack#380]
Is there any equivalent to navigable tasks like that in ResearchKit?

**Comment by jdkizer9** [comment]
@luckysharms9, check out NavigableOrderedTask.java on the develop branch. @liujoshua probably has a better idea of the state that this is in.

**Comment by luckysharms9** [comment]
I'm trying to integrate it into my current project, however there are some instances where it says "Default constructor needed for serialization/deserialization of the object" { super() }, and this method cannot be resolved. It is also trying to find an instance of the method "deepCopy"

I added the following files: NavigationExpectedAnswerQuestion, NavigationFormStep, Navigation QuestionStep, SubstepListStep, SubtaskStep, NavigableOrderedTask, AlertFrameLayout, StepHelper, StepResultHelper, ViewPagerSubstepListStep

**Comment by liujoshua** [comment]
@luckysharms9, it sounds like you copied over NavigableOrderedTask and related classes? Is this what you meant by integrate?

**Comment by luckysharms9** [comment]
Yes, that's what I did. I need researchstack to be local to the project and not pulled as an API because I'm changing some of the files and functionality of it.

**Comment by liujoshua** [comment]
The develop branch is work in progress for a new major release. The changes are quite significant so, unfortunately, I think it'll take a significant amount of work to get a mix of the two versions to work together.
